debug/test/browser/devtools.test.js

```diff
@@ -278,19 +285,29 @@ describe('devtools', () => {
 
 	describe('getChildren', () => {
 		it('should get component children', () => {
-			let a = createElement('div', { foo: 1 });
+			const Foo = () => <div>foo{null}bar</div>;
```

Comment:
@marvinhagemeister Can you double check that my updates here make sense and stay true to the original intent? I quickly tapped around the devtools with these changes and it seems everything worked

Reply:
Clicked around for a while but I can't find any new issues with the devtools :+1: